doc/source/parameters/cfd/physical_properties.rst

```diff
@@ -246,12 +258,14 @@ The ``tracer diffusivity model`` parameter sets which diffusivity model is used.
     subsection physical properties
       set number of fluids = 1
       subsection fluid 0
-        set kinematic viscosity = 0.01
+        set kinematic viscosity      = 0.01
```

Comment:
why this change here?

Reply:
When I ran prmindent on the section, it moved to align to `set tracer diffusivity model = immersed solid tanh` below